docs/api/frame/replace.rst

```diff
@@ -29,24 +29,27 @@
 
     Parameters
     ----------
-    replace_what: None, bool, int, float, list, or dict
+    replace_what: None | bool | int | float | str | list | dict
```

Comment:
I guess you need to `git merge main` on your branch, so that these changes are not showing up.